feedback/lib/src/theme/feedback_theme.dart

```diff
@@ -32,14 +36,28 @@ class FeedbackThemeData {
     this.drawColors = _defaultDrawColors,
     this.bottomSheetDescriptionStyle = _defaultBottomSheetDescriptionStyle,
     this.sheetIsDraggable = true,
-  }) :
+  })  :
         // if the user chooses to supply custom drawing colors,
         // make sure there is at least on color to draw with
         assert(
           // ignore: prefer_is_empty
           drawColors.length > 0,
           'There must be at least one color to draw with',
-        );
+        ),
+        brightness = ThemeData.estimateBrightnessForColor(feedbackSheetColor);
+
+  /// Create a dark version of the [FeedbackThemeData]
+  factory FeedbackThemeData.dark({bool sheetIsDraggable = true}) =>
```

Comment:
We could have one definitely but it will be the equivalent of the default constructor (`FeedbackThemeData()`). But you are right, for consistency, there should be a `light` version. Let me add it!